fcomm/examples/Makefile

```diff
@@ -16,29 +16,41 @@ fibonacci-proof2.json : fibonacci-claim.json
 fibonacci-proof : fibonacci-proof.json
 fibonacci-proof2 : fibonacci-proof2.json
 
-verify-fibonacci-proof :
+verify-fibonacci-proof : fibonacci-proof.json
 	cargo run --release -- verify --proof fibonacci-proof.json
 
-num-list-commitment.json : num-list-function.json
+verify-fibonacci-proof2 : fibonacci-proof2.json
+	cargo run --release -- verify --proof fibonacci-proof2.json
+
+num-list-commitment : num-list-function.json
 	cargo run --release -- commit --function num-list-function.json --commitment num-list-commitment.json
 
-num-list-input.lurk : num-list-input.lurk
+num-list-opening : num-list-opening.json
 
-num-list-opening.json : num-list-function.json num-list-input.lurk
+num-list-opening.json : num-list-function.json num-list-input.lurk num-list-commitment
```

Comment:
How is `num-list-commitment` necessary for this?

Reply:
The commitment file is not necessary, but the commitment needs to have been created so it is 'known to the system'. Otherwise we won't be able to recover the (secret) source when opening by commitment identity later.